ray-operator/test/e2e/testdata/rayservice.autoscaling.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: test-rayservice
+spec:
+  serveConfigV2: |
+    proxy_location: EveryNode
```

Comment:
Does this Ray Serve application have `autoscaling_config` set?https://docs.ray.io/en/latest/serve/api/doc/ray.serve.config.AutoscalingConfig.html#ray-serve-config-autoscalingconfig

Reply:
Done